File: CourierAmexClient/src/app/features/company/models/currency.model.ts
import { BaseEntity } from "@app/models";

export interface CurrencyModel extends BaseEntity<number> {
  companyId?: number;
  companyName?: string;
  name: string;
  buyingRate?: number;
  sellingRate?: number;
  eXRDate?: number;
}


export const newCurrency = {
  companyId:0,
  id: 0,
  name: '',
  status: 2,
  buyingRate: 0,
  sellingRate: 0,
  eXRDate: 0
};

File: CourierAmexAPI/CourierAmex/Models/CurrencyModel.cs
namespace CourierAmex.Models
{
    public class CurrencyModel: BaseEntityModel<int>
    {
        public required int CompanyId { get; set; }
        // CODE
        public required string Name { get; set; }
        // DESCRIPTION
        public decimal BuyingRate { get; set; }
        
        public decimal SellingRate { get; set; }

        public DateTime EXRDate { get; set; }

    }
}
